lib/tasks/pulsearch.rake

```diff
@@ -29,4 +50,18 @@ namespace :pulsearch do
       solr.update data: '<commit/>'
     end
   end
+
+  private
+
+    def copy_libs(lib_dir, instance_dir)
+      dest_dir = File.join(instance_dir,'contrib','analysis-extras',lib_dir)
+      Dir.glob("#{Rails.root.join('solr', lib_dir)}/*.jar").each do |file|
+        dest = File.join(dest_dir, File.basename(file))
+        cp(file, dest) unless File.exists?(dest)
+      end
+    end
+
+    def url_for_file(file)
+      "https://raw.githubusercontent.com/pulibrary/pul_solr/master/configs/orangelight/#{file}"
```

Comment:
i changed the name of `configs` to `solr_configs` to distinguish the solr config from the capistrano config.